### DISCUSSION TITLE
wire-test-utils no longer published?

### DISCUSSION THREAD

**Discussion by martypitt** [square/wire#2194] [Q&A] [answered]
I'm trying to leverage the `RepoBuilder` class for some tests I'm writing, but it appears that `wire-test-utils` hasn't been published to [maven central](https://mvnrepository.com/artifact/com.squareup.wire/wire-test-utils) since 4.0.0.alpha-7.

Is there a reason this has been removed? Has the library been deprecated?

**Comment by JakeWharton** [comment, marked as answer]
I suspect it was only ever published in error as it's internal testing utilities and does not have a stable API.

**Reply by martypitt** [reply]
Thanks for the answer.  That's a shame, as the `RepoBuilder` is super helpful for testing, but I appreciate the overhead if people start depending on it.

**Reply by oldergod** [reply]
It's only one file, you can copy its content over to your repo

**Reply by martypitt** [reply]
Yep, fair call.  I've done just that.

Thanks for this project BTW - makes working with Proto a real pleasure.
